torch/csrc/autograd/python_variable.cpp

```diff
@@ -1828,7 +1835,7 @@ c10::intrusive_ptr<TensorImpl> concrete_detach_fn(const c10::impl::PyInterpreter
     args.ptr(),
     kwargs.ptr(),
     "detach",
-    py::module::import("torch").attr("ops").attr("aten").attr("detach").ptr(),
+    py::module::import("torch").attr("ops").attr("aten").attr("detach").attr("default").ptr(),
```

Comment:
huh, I'm surprised this works (granted, I didn't read the opoverload packet PR in detail haha). Does this mean that default is now a reserved keyword for overload names? Do we have tests that ensure that no one tries to name an overload default?

When I look at tools/codegen, I don't see any according logic:

```
ezyang-mbp:pytorch-tmp ezyang$ git grep \"default tools/codegen
tools/codegen/dest/register_dispatch_key.py:    // so this "default" kernel doesn't actually handle backends that don't support at::empty
ezyang-mbp:pytorch-tmp ezyang$ git grep \'default tools/codegen
tools/codegen/gen.py:            arg['default'] = cpp_a.default
tools/codegen/gen.py:        arg['default'] = pythonify_default(cpp.default_expr(a.default, a.type))
tools/codegen/gen.py:        'default': str(f.has_composite_kernel or has_autogenerated_composite_kernel(f))
```

Reply:
@ezyang I added checks in the schema parser to disable default as an overload name in the schema parser to also disable that when registering through the RegisterOperators API https://github.com/pytorch/pytorch/pull/72206/files

Reply:
OK, I recommend also synchronizing the parser in model.py to also have the same checking; otherwise you get the error only at runtime when you load torch. Doesn't have to be this PR.